src/Components/CreateImageWizardV2/steps/Details/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+
+import {
+  Form,
+  FormGroup,
+  FormHelperText,
+  HelperText,
+  HelperTextItem,
+  Text,
+  Title,
+} from '@patternfly/react-core';
+
+import { ValidatedTextInput } from '../../ValidatedTextInput';
+import {
+  isBlueprintDescriptionValid,
+  isBlueprintNameValid,
+} from '../../validators';
+
+const DetailsStep = () => {
+  const [name, setName] = React.useState('');
+  const [description, setDescription] = React.useState('');
+
+  const handleNameChange = (
+    _event: React.FormEvent<HTMLInputElement>,
+    name: string
+  ) => {
+    setName(name);
+  };
+  const handleDescriptionChange = (
+    _event: React.FormEvent<HTMLInputElement>,
+    name: string
+  ) => {
+    setDescription(name);
+  };
```

Comment:
Here the name and the description get stored in the variables that will not persist when continuing to the Review step. What will need to be done is adding both to the wizardSlice and storing them in the store. 

That would mean adding a structure for the step, maybe something like:
```typescript
  details: {
    blueprintName: ComposeRequest["image_name"] | undefined;
    blueprintDescription: ComposeRequest["image_description"] | undefined;
  };
```

and `changeBlueprintName` and `changeBlueprintDescription` functions will be needed. :)

Reply:
thank you @regexowl